src/main/java/edu/iastate/ece/sd/sdmay2126/runner/selenium/FBASeleniumInputProgrammer.java

```diff
@@ -75,8 +75,64 @@ public void programInputs(Job job, WebElement scopedFBACard) throws SeleniumIden
+        WebElement carbUp = scopedFBACard
+                .findElement(By.cssSelector("div[data-parameter='max_c_uptake']"))
+                .findElement(By.cssSelector("input[class='form-control']"));
+        carbUp.clear();
+        carbUp.sendKeys(Float.toString(params.getMaxCarbonUptake()));
+
+        //Finds clears and sets the value of the maxNitrogenUptake value
+        System.out.println("Setting max nitrogen uptake...");
+        WebElement nitroUp = scopedFBACard
+                .findElement(By.cssSelector("div[data-parameter='max_n_uptake']"))
+                .findElement(By.cssSelector("input[class='form-control']"));
+        nitroUp.clear();
+        nitroUp.sendKeys(Float.toString(params.getMaxNitrogenUptake()));
+
+        //Finds clears and sets the value of the maxPhosphateUptake value
+        System.out.println("Setting max phosphate uptake...");
+        WebElement phosUp = scopedFBACard
+                .findElement(By.cssSelector("div[data-parameter='max_p_uptake']"))
+                .findElement(By.cssSelector("input[class='form-control']"));
+        phosUp.clear();
+        phosUp.sendKeys(Float.toString(params.getMaxPhosphateUptake()));
+
+        //Finds clears and sets the value of the maxSulfurUptake value
+        System.out.println("Setting max sulfur uptake...");
+        WebElement sulfUp = scopedFBACard
+                .findElement(By.cssSelector("div[data-parameter='max_s_uptake']"))
+                .findElement(By.cssSelector("input[class='form-control']"));
+        sulfUp.clear();
+        sulfUp.sendKeys(Float.toString(params.getMaxSulfurUptake()));
+
+        //Finds clears and sets the value of the maxOxygenUptake value
+        System.out.println("Setting max oxygen uptake...");
+        WebElement oxyUp = scopedFBACard
+                .findElement(By.cssSelector("div[data-parameter='max_o_uptake']"))
+                .findElement(By.cssSelector("input[class='form-control']"));
+        oxyUp.clear();
+        oxyUp.sendKeys(Float.toString(params.getMaxOxygenUptake()));
+
+        //Finds clears and sets the value of the expressionThreshold value
+        System.out.println("Setting expression threshold...");
+        WebElement expThresh = scopedFBACard
+                .findElement(By.cssSelector("div[data-parameter='exp_threshold_percentile']"))
+                .findElement(By.cssSelector("input[class='form-control']"));
+        expThresh.clear();
+        expThresh.sendKeys(Float.toString(params.getExpressionThreshold()));
+
+        //Finds clears and sets the value of the expressionUncertainty value
+        System.out.println("Setting expression uncertainty...");
+        WebElement expUncer = scopedFBACard
+                .findElement(By.cssSelector("div[data-parameter='exp_threshold_margin']"))
+                .findElement(By.cssSelector("input[class='form-control']"));
+        expUncer.clear();
+        expUncer.sendKeys(Float.toString(params.getExpressionUncertainty()));
+
```

Comment:
All of these are reused code.

Could create a method to extract this logic. The only things changing with each instance is the css selector and the params that get filled in.

Could be something similar to:
```
public void fillInFormWithValueByCSSSelector(String cssSelector, String valueToFill) {
 // body here
}
```